docker-compose.yml

```diff
@@ -1,28 +0,0 @@
-version: '3.7'
```

Comment:
This can come back. I think I have figured out that the trace SUITE is flaky when combined with other tests.